components/azavea/sampleViz.jsx

```diff
@@ -0,0 +1,42 @@
+import Renderer from '@widget-editor/renderer';
+import {giniConfig} from './customChart';
```

Comment:
The non-widget-editor config can be used by importing `fishConfig` instead here.